http_server/routes/socketio.go

```diff
@@ -130,6 +130,15 @@ func getStationOverviewData(stationName string, h *server.Handlers) (models.Stat
 		return models.StationOverviewData{}, err
 	}
 
+	schema, err := h.Schemas.GetSchemaByStationName(station.Schema.SchemaName)
```

Comment:
you don't need to get the schema details since what you need is already saved in the station, just return the name + version + a flag